src/licensedcode/data/licenses/osetpl-2.1.yml

```diff
@@ -0,0 +1,7 @@
+key: osetpl-2.1
+short_name: OSET 2.1
+name: OSET Public License v2.1
+
+category: Proprietary Free
```

Comment:
@yasharmaster Heather authored this license. So please use the same category as for the MPL 2.0